packages/@aws-cdk/cloudformation-include/test/test-templates/functions-and-conditions.json

```diff
@@ -0,0 +1,122 @@
+{
+    "Mappings" : {
+      "RegionMap" : {
+        "region-1" : { 
+          "HVM64" : "name1", "HVMG2" : "name2" 
+        }
+      }
+    },
+    "Conditions": {
+        "AlwaysTrueCond": {
+          "Fn::Not": [{
+            "Fn::Equals": [{"Ref": "AWS::Region"}, "completely-made-up-region"]}]
+        },
+        "AlwaysFalseCond": {
+          "Fn::Equals": [{ "Ref": "AWS::Region" }, "completely-made-up-region"]
+        },
+        "AndCond": {
+          "Fn::And": [
+            {"Condition": "AlwaysTrueCond"},
+            {"Condition": "AlwaysFalseCond"}
+          ] 
+        },
+        "OrCond": {
+          "Fn::Or": [
+            {"Condition": "AlwaysTrueCond"},
+            {"Condition": "AlwaysFalseCond"}
+          ] 
+        }
+      },
+    "Resources": {
+      "Vpc": {
+        "Type" : "AWS::EC2::VPC",
+        "Properties" : {
+          "CidrBlock" : {
+              "Fn::If": [
+                  "AlwaysTrueCond",
+                  { "Fn::Cidr": [ "192.168.1.1/24", 2] },
+                  { "Fn::Cidr": [ "10.0.0.0/24", 6, 5] }
+              ]
+          }
+        }
+      },
+      "Bucket": {
+        "Type": "AWS::S3::Bucket",
+        "Properties": {
+          "BucketName": {
+            "Fn::If": [
+              {"Fn::Or": ["AlwaysFalseCond", "AlwaysFalseCond"]},
+              "Unreachable",
+              {"Fn::FindInMap": [
```

Comment:
Don't use the shorthand of opening brace and the only key of an an object if it that key contains a complex value (= anything other than a string) - otherwise, it looks weird. So here, it should be:

```json
{
  "Fn::FindInMap": [
    "RegionMap",
    "region-1",
    "HVM64"
   ]
}
```

Reply:
Let me slightly rephrase that :). "Don't use the shorthand if the value is not on the same line". So you could also do (notice the spaces around `{` and `}`):

```json
{ "Fn::FindInMap": ["RegionMap", "region-1", "HVM64"] },
```